src/ontology/Makefile

```diff
@@ -388,6 +388,16 @@ $(IMPORTDIR)/obi_import.owl: $(MIRRORDIR)/obi.owl $(IMPORTDIR)/obi_terms_combine
 		query --update ../sparql/inject-subset-declaration.ru --update ../sparql/inject-synonymtype-declaration.ru --update ../sparql/postprocess-module.ru \
 		$(ANNOTATE_CONVERT_FILE); fi
 
+## Module for ontology: cob
+
+$(IMPORTDIR)/cob_import.owl: $(MIRRORDIR)/cob.owl $(IMPORTDIR)/cob_terms_combined.txt
+	if [ $(IMP) = true ]; then $(ROBOT) query -i $< --update ../sparql/preprocess-module.ru \
+    extract -T $(IMPORTDIR)/cob_terms_combined.txt --copy-ontology-annotations true --force true --method BOT \
+    remove --base-iri $(OBOBASE)/COB --axioms external --preserve-structure false --trim false \
```

Comment:
I think you will have to use "custom" module type here, because this row

```
    remove --base-iri $(OBOBASE)/COB --axioms external --preserve-structure false --trim false \
```

Is wrong for COB (and only for COB).

Reply:
copied the same rule without this line to ro.Makefile

Reply:
So this means now we can import 3rd party ontology terms via COB?

Reply:
Yes.